visualize-admin/visualization-tool
# Feat/homepage add ons

This PR resolves #1808 ,resolves #1827 and #1828
- Introduces a Section for Newsletter subscriptions 
- Introduces a Section for Bug Reports
- Introduces a Section for Feature Requests
- Allows Bug Reports and Feature Requests on several pages 

**What Still needs to be Improved**
- The design can be improved for the Bug Report 
- The design for the Newsletter can not be implemented just yet because it would look weird due to the fact the entire landing page still follows the old design.

### How to Test
1. Go to this [Link](https://vercel.live/open-feedback/visualization-tool-git-feat-homepage-add-ons-ixt1.vercel.app?via=pr-comment-visit-preview-link&passThrough=1) 
2. Scroll to the bottom of the page 
3. See how the [Design](https://www.figma.com/design/Hn9wvuEYLUmdJtg4KwZYUx/visualize.admin.ch---design?node-id=5850-44207&t=moajWfo6OPOxdE2X-4) from @KerstinFaye is implemented ✅
4. Click on subscribe or Report a bug See how a `mailto` tab opens w/ the recipient being BAFU ✅

---

### Next Steps (for 1808 & 1802)
1. Add the selected ixt email
2. Add preferred body based on language
3. Add preferred subject 

---

### What will be included in the next PR
- Feedback button for Users and non Users in the header

CC: @KerstinFaye , @sosiology
